src/services/user-bff/src/routes/stripe.ts

```
@@ -0,0 +1,115 @@
+      return handleCancelled(event, c);
+    case "payment_intent.payment_failed":
+      return handleFailed(event, c);
+    case "payment_intent.processing":
+      return handleProcessing(event, c);
+    default:
+      console.info(
+        "Stripe event triggered but no action was taken",
+        event.type,
+      );
+      return c.text("no action taken", 200);
+      break;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)